.github/ISSUE_TEMPLATE/epic.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Epic
+about: Create an epic to plan a major new initiative
+title: ''
+labels: Epic, kind/epic
+assignees: ''
+
+---
+
+# _Name of the Epic_
+
+_Provide a short description of the epic here._
+
+## References
+- [_Aha Card Title_](https://cyberark.aha.io/features/)
+- [Feature Doc]()
+
+## Team
+- **Product Manager**: _Persons Name_ (@username)
+- **Product Owner**: _Persons Name_ (@username)
+- **Engineering Manager**: _Persons Name_ (@username)
+- **Feature Lead**: _Persons Name_ (@username)
+- **Engineers**:
+    - _Persons Name_ (@username)
+- **Technical Writer**: _Persons Name_ (@username)
+
```

Comment:
I ditched the big table of SDLC dates.  I don't feel like any of us use/update them.

Reply:
Agree, and we're tracking that stuff in a spreadsheet now.